estate/models/estate_property.py

```diff
@@ -0,0 +1,63 @@
+from dateutil.relativedelta import relativedelta
+
+from odoo import api, models, fields
+
+
+class EstateProperty(models.Model):
+    _name = 'estate.property'
+    _description = 'real estate property model'
+
+    name = fields.Char(required=True)
+    active = fields.Boolean(default=True)
+    description = fields.Text()
+    postcode = fields.Char()
+    date_availability = fields.Date(copy=False, default=lambda x: fields.Datetime.today() + relativedelta(months=+3))
+    expected_price = fields.Float(required=True)
+    selling_price = fields.Float(readonly=True, copy=False)
+    bedrooms = fields.Integer(default=2)
+    living_area = fields.Integer()
+    facades = fields.Integer()
+    garage = fields.Boolean()
+    garden = fields.Boolean()
+    garden_area = fields.Integer()
+    total_area = fields.Integer(compute='_compute_total_area')
+    garden_orientation = fields.Selection([
+        ('north', 'North'),
+        ('south', 'South'),
+        ('east', 'East'),
+        ('west', 'West')
+    ])
```

Comment:
In python, for quoting strings we try to follow the following convention:
- single quote around technical terms
- double quotes around display terms

=> `('north', "North")`

Let's try to follow this during this tutorial, even though you'll find many counter examples in the existing code.